bibliothecae-it.csl

```diff
@@ -0,0 +1,246 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="it-IT">
+  <info>
+    <title>Bibliothecae.it</title>
+    <id>http://www.zotero.org/styles/bibliothecae-it</id>
+    <link href="http://www.zotero.org/styles/bibliothecae-it" rel="self"/>
+    <link href="http://www.zotero.org/styles/einaudi" rel="template"/>
+    <link href="https://bibliothecae.unibo.it/about/submissions#authorGuidelines" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien</name>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="humanities"/>
+    <issn>2283-9364</issn>
+    <updated>2021-05-14T07:32:38+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="it">
+    <terms>
+      <term name="page" form="short">
+        <single>p.</single>
+        <multiple>pp.</multiple>
+      </term>
+      <term name="and others"/>
+      <term name="letter" form="short">lettera</term>
+    </terms>
+  </locale>
+  <macro name="responsibility">
+    <names variable="author">
+      <name delimiter-precedes-last="never" et-al-min="4" et-al-use-first="4"/>
```

Comment:
et-al-min needs to be > et-al-use-first for logical reasons. Do you want 4/3 here or 4/1 as below?

Reply:
thanks for spotting that error!